dev/com.ibm.ws.security.fat.common.backchannelLogout/fat/src/com/ibm/ws/security/backchannelLogout/fat/CommonTests/LogoutTokenValidationTests.java

```diff
@@ -599,6 +628,133 @@ public void LogoutTokenValidationTests_invalid_iat_1971() throws Exception {
         genericAddedUpdatedClaimsTest(Constants.PAYLOAD_ISSUED_AT_TIME_IN_SECS, 36010452);
 
     }
+    
+    /**
+     * Test that the backchannelLogout fails when we omit the required exp claim from the logout token.
+     *
+     * @throws Exception
+     */
+    @Test
+    public void LogoutTokenValidationTests_missing_required_exp() throws Exception {
+
+        genericMissingRequiredClaimTest(Constants.PAYLOAD_EXPIRATION_TIME_IN_SECS);
+
+    }
+    
+    /**
+     * Test that the backchannelLogout fails when we specify an exp that is before the iat
+     *
+     * @throws Exception
+     */
+    @Test
+    public void LogoutTokenValidationTests_invalid_exp_before_iat() throws Exception {
+
+    	long iat = System.currentTimeMillis() / 1000;
+        long exp = iat - hoursToSeconds(1);
+
+        genericInvalidIatOrExpClaimTest(iat, exp);
+
+    }
+
+    /**
+     * Test that the backchannelLogout fails when we specify an exp that is way in the past
+     *
+     * @throws Exception
+     */
+    @Test
+    public void LogoutTokenValidationTests_invalid_exp_1971() throws Exception {
+
+        long exp = 36010452; // 02/21/1971
+        long iat = exp - minutesToSeconds(2);
+
+        genericInvalidIatOrExpClaimTest(iat, exp);
+
+    }
+
+    /**
+     * Test that the backchannelLogout fails when we specify an exp that is from yesterday
+     *
+     * @throws Exception
+     */
+    @Test
+    public void LogoutTokenValidationTests_invalid_exp_yesterday() throws Exception {
+
+        long exp = System.currentTimeMillis() / 1000 - hoursToSeconds(24);
+        long iat = exp - minutesToSeconds(2);
+
+        genericInvalidIatOrExpClaimTest(iat, exp);
+
+    }
+
+    /**
+     * Test that the backchannelLogout fails when we specify an exp that is just in the past (just beyond the clockskew)
+     *
+     * @throws Exception
+     */
+    @Test
+    public void LogoutTokenValidationTests_invalid_exp_past_beyond_clockSkew() throws Exception {
+
+        long clockSkew = 5;
+        long exp = System.currentTimeMillis() / 1000 - minutesToSeconds(clockSkew);
+        long iat = exp - minutesToSeconds(2);
+
+        genericInvalidIatOrExpClaimTest(iat, exp);
+
+    }
+
+    /**
+     * Test that the backchannelLogout passes when we specify an exp that is the current time (due to the clowSkew)
+     *
+     * @throws Exception
+     */
+    @Test
+    public void LogoutTokenValidationTests_invalid_exp_now() throws Exception {
```

Comment:
This should be a valid case, yes?
```suggestion
    public void LogoutTokenValidationTests_valid_exp_now() throws Exception {
```